etc/picongpu/lumi-eurohpc/lumi-G_hipcc_picongpu.profile.example

```diff
@@ -0,0 +1,147 @@
+printf "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n" >&2
+printf "@ Set the initial buffer size in your ADIOS2              @\n" >&2
+printf "@ configuration of your job's *.cfg file to 28GiB,        @\n" >&2
```

Comment:
This message can be removed because we workaround the HIP issue that the reported free memory is wrong.

Reply:
@psychocoderHPC is fixed now